ui/src/bichat/hooks/useMarkdownCopy.ts

```
@@ -0,0 +1,133 @@
+    const timeout = timeoutsRef.current.get(blockId)
+    if (timeout) {
+      clearTimeout(timeout)
+      timeoutsRef.current.delete(blockId)
+    }
+  }, [])
+
+  const resetAll = useCallback(() => {
+    setCopiedStates(new Map())
+
+    // Clear all timeouts
+    timeoutsRef.current.forEach((timeout) => clearTimeout(timeout))
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)